airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/jdbc/DatabricksSqlGenerator.kt

```diff
@@ -82,18 +83,20 @@ class DatabricksSqlGenerator(
         name: String,
         rawNamespaceOverride: String
     ): StreamId {
+        // Databricks downcases all object names, so handle that here
         return StreamId(
-            namingTransformer.getNamespace(namespace),
-            namingTransformer.getIdentifier(name),
-            namingTransformer.getNamespace(rawNamespaceOverride),
-            namingTransformer.getIdentifier(StreamId.concatenateRawTableName(namespace, name)),
+            namingTransformer.getNamespace(namespace).lowercase(),
+            namingTransformer.getIdentifier(name).lowercase(),
+            namingTransformer.getNamespace(rawNamespaceOverride).lowercase(),
+            namingTransformer.getIdentifier(StreamId.concatenateRawTableName(namespace, name)).lowercase(),
```

Comment:
This is where namingTransformer would've really helped if we had the methods separation correctly 😅   unfortunately getIdentifier is shared between columns and raw table name.

Reply:
yuuup. though it's kind of redundant with sqlgenerator making that split anyway :shrug: there's probably a world where we kill NamingTransformer and just do everything directly in sqlgenerator